assets/app/views/create/nextSteps.html

```diff
@@ -0,0 +1,102 @@
+      </div>
+
+      <div class="buttons" ng-if="!tasks().length">
+        <a class="btn btn-primary btn-lg" href="project/{{projectName}}/overview">Go to overview</a>
+      </div>
+
+      <div ng-repeat="task in tasks()">
+        <h3>
+          <i class="spinner spinner-xs spinner-inline" ng-show="task.status=='started'"></i>
+          {{ task | taskTitle }}.
+        </h3>
+
+        <div ng-repeat="alert in task.alerts">
+          <div ng-switch="alert.type">
+            <div ng-switch-when="error">
+              <span class="pficon-layered">
+                <span class="pficon pficon-error-octagon"></span>
+                <span class="pficon pficon-error-exclamation"></span>
+              </span>
+              <span ng-if="alert.message">{{alert.message}}</span><span ng-if="alert.details">{{alert.details}}.</span>
+            </div>
+            <div ng-switch-when="warning">
+              <span class="pficon-layered">
+                <span class="pficon pficon-warning-triangle"></span>
+                <span class="pficon pficon-warning-exclamation"></span>
+              </span>
+              <span ng-if="alert.message">{{alert.message}}</span><span ng-if="alert.details">{{alert.details}}.</span>
+            </div>
+            <div ng-switch-when="success">
+              <span class="pficon pficon-ok"></span>
+              <span ng-if="alert.message">{{alert.message}}</span><span ng-if="alert.details">{{alert.details}}</span>
+            </div>  
+          </div>
+        </div>
+
+        <div class="buttons gutter-top" ng-if="!task.hasErrors">
+          <a class="btn btn-primary btn-lg" href="project/{{projectName}}/overview">Go to overview</a>
+        </div>
+
+      </div>
+    </div>
+
+    <div class="row" ng-controller="TasksController">
+      <div ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length" class="col-md-12">
+        <h2>Things you can do</h2>
+        <p>Go to the <a href="project/{{projectName}}/overview">overview page</a> to see more details about this project. Make sure you don't already have <a href="project/{{projectName}}/browse/services">services</a>, <a href="project/{{projectName}}/browse/builds">build configs</a>, <a href="project/{{projectName}}/browse/deployments">deployment configs</a>, or other resources with the same names you are trying to create. Refer to the <a target="_blank" href="{{'new_app' | helpLink}}">documentation for creating new applications</a> for more information.</p>
+
+        <h3>Command line tools</h3>
+        <p>You may want to use the <code>oc</code> command line tool to help with troubleshooting. After <a target="_blank" href="{{'get_started_cli' | helpLink}}">downloading and installing</a> it, you can log in, switch to this particular project, and try some commands :</p>
+
+        <pre class="code prettyprint">oc login {{baseUrl}}
```

Comment:
you pulled baseUrl off of location.origin, you can't assume the hostname of the console is the same as the hostname of the api.  Add something to DataService that will return the hostPort of the openshift api.  It reads it from what it gets back in config.js, see resourceInfo method to see how to get it off existing objects on data service.